src/init.cpp

```diff
@@ -228,7 +228,6 @@ void PrepareShutdown()
 #endif
     GenerateBitcoins(false, 0, Params(), *g_connman);
```

Comment:
If g_connman has not yet been created here won't this dereference also cause a crash ?